# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.h
# Commit: c0a7fdc4bfca
# Full Hash: c0a7fdc4bfca0596ffecd347f0e248440e371989
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P5: Unify GetLevelInternal and GetDefaultLevel as Accessible::GetLevel. r=Jamie
#   
#   The nsAccUtils method was a "fast" one for calculating set sizes and
#   conceptual parents. Unified it with
#   LocalAccessible::GetLevelInternal in Accessible::GetLevel with an
# ==============================================================================

diff -r f6ce0e05c92e -r c0a7fdc4bfca accessible/generic/LocalAccessible.h
--- a/accessible/generic/LocalAccessible.h	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/generic/LocalAccessible.h	Tue Jan 04 21:01:36 2022 +0000
@@ -272,11 +272,6 @@
   virtual LocalAccessible* FocusedChild();
 
   /**
-   * Return calculated group level based on accessible hierarchy.
-   */
-  virtual int32_t GetLevelInternal();
-
-  /**
    * Calculate position in group and group size ('posinset' and 'setsize') based
    * on accessible hierarchy.
    *